render.yaml

```diff
@@ -16,10 +22,9 @@ services:
         fromDatabase:
           name: hostedgpt
           property: connectionString
-      - key: RAILS_MASTER_KEY
+      - fromGroup: generated-secrets
```

Comment:
This is cool! Can you please also update the README.md file to remove that step of copy/pasting the key?

Reply:
Good call. I updated the instructions there a little. We should consider whether forking the repository is necessary for a render deploy -- if not, we can make the process even easier.